lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUCloseTaskTest.cs

```diff
@@ -53,7 +53,7 @@ public void TestTaskCloseOnLocalRuntime()
         {
             const int chunkSize = 2;
             const int dims = 50;
-            const int iterations = 200;
+            const int iterations = 1000;
```

Comment:
Task could still fail by communication exception before receiving close event, depending on if it receives close event earlier or getting communication error cause by other tasks' close earlier.